global/matlab/icaDim/icaDim.m

```diff
@@ -176,10 +176,6 @@
     %Maximum of laplacian is estimate of optimal dimensionality
     [~, Out.calcDim]=max(Out.pcaDim_lambdaAdj.lap(1:DOF));
     
-    if VN~=1
-      stabCount = 2;
-    end
-    
```

Comment:
I saw this code, but are you sure it can be deleted?  After all, `stabCount` is a variable used elsewhere, so I assumed that there was a reason for these 3 lines...

Reply:
It is unconditionally assigned to below the display block, with no intervening flow control or uses:

https://github.com/Washington-University/HCPpipelines/blob/fcef464fb50e8ec4787f5a11538ba71a263b29d5/global/matlab/icaDim/icaDim.m#L200

If it did have an effect on the loop control, it would already cause a premature exit under strange conditions.

Reply:
As for a reason, it was probably an attempt to force loop exit after a single iteration in the "no normalize" mode (which would actually be a reasonable feature, but I don't think it did that), without considering the VN > 1 or stabThresh = 3 cases (some of which may not have been implemented at the time), and then was forgotten and never cleaned up.